app/models/bulkrax/importer.rb

```diff
@@ -74,6 +74,10 @@ def import_collections
       parser.create_collections
     end
 
+    def add_parent_child_relationships
+      parser.create_parent_child_relationships
```

Comment:
Is it better to do `importer.create_parent_child_relationships` (note the create) and then delegate it to the parser?